scene/animation/animation_node_state_machine.h

```diff
@@ -288,6 +288,8 @@ class AnimationNodeStateMachinePlayback : public Resource {
 
 	bool is_grouped = false;
 
+	void _clear_fade_and_maybe_emit_finished(AnimationNodeStateMachine *p_state_machine, const StringName p_state);
```

Comment:
```suggestion
	void _clear_fading(AnimationNodeStateMachine *p_state_machine, const StringName p_state);
```

No need to be that much specific about the method name. Also, as I've said elsewhere, avoid ambiguous words like "maybe".